packages/amplify-provider-awscloudformation/src/graphql-transformer/transform-config.ts

```diff
@@ -0,0 +1,196 @@
+import * as path from 'path';
+import { TransformConfig, TransformerProjectConfig } from '@aws-amplify/graphql-transformer-core';
+import fs from 'fs-extra';
+
+export interface ProjectOptions {
+  projectDirectory?: string;
+  transformersFactory: Function;
+  transformersFactoryArgs: object[];
+  currentCloudBackendDirectory: string;
+  rootStackFileName?: string;
+  dryRun?: boolean;
+  disableFunctionOverrides?: boolean;
+  disablePipelineFunctionOverrides?: boolean;
+  disableResolverOverrides?: boolean;
+  buildParameters?: Object;
+  minify?: boolean;
+}
+
+
+export const TRANSFORM_CONFIG_FILE_NAME = `transform.conf.json`;
+
+/**
+ * try to load transformer config from specified projectDir
+ * if it does not exist then we return a blank object
+ *  */
+
+export async function loadConfig(projectDir: string): Promise<TransformConfig> {
+  // Initialize the config always with the latest version, other members are optional for now.
+  let config: TransformConfig = {
+    schema: ''
+  };
+  try {
+    const configPath = path.join(projectDir, TRANSFORM_CONFIG_FILE_NAME);
```

Comment:
projectDir is the API resource's directory? since I think that is where we have this file.

Reply:
We should use different naming then.